covid_moonshot/analysis/plots.py

```diff
@@ -252,6 +253,40 @@ def plot_convergence(
     return fig
 
 
+def plot_poor_convergence_fe_table(
+    runs: List[int], runs_details: List[int], energy_cutoff: float = 1.0,
+) -> plt.Figure:
+
+    complex_phases = [run.complex_phase for run in runs]
+    job_ids = [run.JOBID for run in runs_details]
+
+    std_dev_store = []
+    jobid_store = []
+
+    for i, run in enumerate(runs):
```

Comment:
We should get the run number from `run.details.run_id()` instead of using `enumerate` here, because `runs` is not necessarily contiguous (e.g. if we skip a run due to not having enough data). Also, then we don't having to worry about maintaining the invariant that `runs` is always sorted by run id.

Reply:
(it doesn't seem like `i` is used below)